packages/@aws-cdk-containers/ecs-service-extensions/README.md

```diff
@@ -322,9 +323,26 @@ const environment = Environment.fromEnvironmentAttributes(stack, 'Environment',
 
 ```
 
+## Injecter Extension
+
+This service extension accepts a list of `Injectable` resources that the service can publish events or write data to and sets up the corresponding permissions for the service.
```

Comment:
I wanna avoid saying publish events specifically in the first sentence. The `Injectable` interface can be implemented by various resource. But we can give an example with `InjectableTopic`.

```suggestion
This service extension accepts a list of `Injectable` resources that grants access to the resources and adds the necessary environment variables to the tasks part of the service.  

For example, an `InjectableTopic` is an SNS topic that grants permission to the task role and adds the topic name as an environment variable to the task definition.
```
feel free to change the wording!